### PR TITLE
Adding post-processing folder to store scripts used to post-process data

### PR DESCRIPTION
**Pull request by lucbv**
Currently it contains a module that parses the log files from Nalu
It extracts data per equation as well as some global data.
Also adding a script showing how to use the module to generate
FY20Q2 milestone weak scaling tables.

### CODE REVIEW

File: post-processing/post_process_nalu.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    fi = open(filename, "r")
+    data = fi.readlines()
+    fi.close()
+
+    # Step 2: count total linear iterations per equation
+    numLinIters = np.zeros(len(equations), dtype=np.int32)
+    times       = np.zeros((len(equations), 8))
+    for lineIdx in range(len(data)):
+        line = data[lineIdx]
+	if("STKPERF: Total Time:" in line):
```
Comment (sayerhs) [inline post-processing/post_process_nalu.py]: Looks like wrong indentation.

Comment (lucbv) [reply post-processing/post_process_nalu.py]: it's actually a tab vs whitespace issue, I will run a command that makes everything whitespaces.

**Comment by lucbv** [comment]
@sayerhs how about know, I ran untabify, it should be OK

**Review by sayerhs — APPROVED** [review]
lgtm

**Pull request merged**